distribution/build.gradle

```diff
@@ -210,6 +211,44 @@ xpack.subprojects.findAll { it.parent == xpack }.each { Project xpackModule ->
   copyLog4jProperties(buildDefaultLog4jConfig, xpackModule)
 }
 
+/*****************************************************************************
+ *                                    JDKs                                   *
+ *****************************************************************************/
+List<String> bundledJdkVersion = VersionProperties.bundledJdk
+String jdkVersion = "${bundledJdkVersion[0..2].join('.')}"
+String jdkMajor = bundledJdkVersion[0]
+String jdkBuild = bundledJdkVersion[3]
+repositories {
+  ivy {
+    url "https://download.java.net"
```

Comment:
I'm worried about this in CI. 
I assume this being a repository it does benefit from caching, and the artifacts will be pulled down by running the `resolveAllDependencies` task. 
As long as we make sure that  this is true I'm fine with the change.

Reply:
Yes, resolveAllDependencies is triggering the download:

> :distribution:resolveAllDependencies > Resolve dependencies of :distribution:jdk_darwin > openjdk-11.0.1_osx-x64_bin.xml